packages/schematics/src/update/update-5.0.0.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import { Rule, Tree, SchematicContext } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import { updateJsonFile } from '../utility/update-json-file';
+
+const clarityCorePackageName = '@clr/core';
```

Comment:
We are about to change this one to `@cds/core` for version 5. I think it is okay for now to push it as it is and change it when Matt update the package name. I think creating a task for it is a good idea. What do you think?

Reply:
Yup, that sounds good!